src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -117,98 +117,166 @@ export class CardItem extends React.Component<DevFileProps, {
+                        {selectedVersion.description &&
+                            <div className={this.props.cardItemStyle.detailedDescription}>
+                                <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Description</Typography>
+                                <Typography variant='body2' className={this.props.cardItemStyle.helmCardDetailItemValue}>
+                                    {selectedVersion.description}
+                                </Typography>
+                            </div>
+                        }
+                        {
+                            selectedVersion.annotations['charts.openshift.io/providerType'] &&
+                            <div className={this.props.cardItemStyle.detailedDescription}>
+                                <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Source</Typography>
+                                <Typography variant='body2' className={this.props.cardItemStyle.helmCardDetailItemValue}>
+                                    {capitalizeFirstLetter(selectedVersion.annotations['charts.openshift.io/providerType'])}
+                                </Typography>
+                            </div>
+                        }
+                        <div className={this.props.cardItemStyle.detailedDescription}>
+                            <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Product Version</Typography>
+                            <Typography variant='body2' className={this.props.cardItemStyle.helmCardDetailItemValue}>
+                                {selectedVersion.appVersion || selectedVersion.version}
+                            </Typography>
                         </div>
+                        {
+                            selectedVersion.annotations['charts.openshift.io/supportURL'] &&
+                            <div className={this.props.cardItemStyle.detailedDescription}>
+                                <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Home Page</Typography>
+                                <Typography variant='body2' className={this.props.cardItemStyle.helmCardDetailItemValue}>
+                                    <a className={this.props.cardItemStyle.helmCardDetailItemValue} href={selectedVersion.annotations['charts.openshift.io/supportURL']}>{selectedVersion.annotations['charts.openshift.io/supportURL']}</a>
+                                </Typography>
+                            </div>
+                        }
+                        {selectedVersion.maintainers &&
+                            <div className={this.props.cardItemStyle.detailedDescription}>
+                                <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Maintainers</Typography>
```

Comment:
The css padding for the modal has to be looked into as the space below the Maintainers div is not getting loaded.

For reference see the image, the padding for the `Maintainers` div is not used.

<img width="1023" alt="Screenshot 2023-05-22 at 1 44 44 AM" src="https://github.com/redhat-developer/vscode-openshift-tools/assets/3133356/83e9bf50-0869-44cc-b222-8d656e50a801">